docs/patterns/disclosure/index.md

```diff
@@ -0,0 +1,247 @@
+
+### Best practices
+
+A disclosure consists of one panel only. If more than one panel is needed, use 
+an accordion instead.
+
+{% example palette="wrong",
+           class="centered",
+           alt="Inline alert in the top right corner of a page",
+           src="./disclosure-bestpractice-1.svg" %}
+
+{% example palette="wrong",
+           class="centered",
+           alt="Inline alert in the top right corner of a page",
+           src="./disclosure-bestpractice-1.svg" %}
+
+Text inside the panel shouldn’t exceed eight grid columns to maintain optimal 
+readability.
+
+{% example palette="wrong",
+           class="centered",
+           alt="Inline alert in the top right corner of a page",
+           src="./disclosure-bestpractice-2.svg" %}
+
+Don’t wrap other complex components inside of a disclosure unless absolutely 
+necessary, like jump links.
+
+{% example palette="wrong",
+           class="centered",
+           alt="Inline alert in the top right corner of a page",
+           src="./disclosure-bestpractice-3.svg" %}
+
+<hgroup>
+
+  ## Behavior
+  ### States
+
+</hgroup>
+
+A disclosure can be collapsed and expanded to hide or reveal information when 
+toggled. By default, it loads with its panel collapsed (this can be customized), 
+allowing users to get a quick preview of the content inside.
+
+### Collapsing and expanding
+
+The disclosure panel can be collapsed or expanded by clicking on or tapping the 
+caret icon, the  section text label, or anywhere inside the container. When the 
+panel is collapsed, the caret icon points to the right toward the text. When the 
+panel expands, the caret icon rotates 90º and points down, revealing the content 
+inside.
+
+{% example palette="light",
+           class="centered",
+           alt="Disclosure with the expand/collapse panel highlighted",
+           src="./disclosure-behavior-1.svg" %}
+
+### Tab order
+
+When the <kbd>Tab</kbd> key is pressed, the focus indicator highlights the 
+collapsed disclosure. When a user expands the panel by pressing the 
+<kbd>Enter</kbd> or <kbd>Space</kbd> keys, any interactive elements inside are 
+added to the tab order before the focus indicator moves away to the next section 
+of content.
+
+{% example palette="light",
+           class="centered",
+           alt="Disclosure showing links with the focus style",
+           src="./disclosure-behavior-2.svg" %}
+
+## Responsive design
+
+A disclosure works well when used on both large and small screens.
+
+### Breakpoints
+
+Disclosures can be used on smaller screens, but the limited screen space makes 
+content taller which might make users scroll more, so be careful about how much 
+content is included.
+
+### Desktop
+
+![Example of a disclosure on desktop](./disclosure-responsive-1.svg)
+
+### Tablet
+
+![Example of a disclosure on tablet](./disclosure-responsive-2.svg)
+
+### Mobile
+
+![Example of a disclosure on mobile](./disclosure-responsive-3.svg)
```

Comment:
We should use the example shortcode here to ensure proper image sizing.

Reply:
Spun out for its own issue, issue with example shortcode and svgs that and double scaled